site/content/docs/developing/addons/modeling.en.md

```diff
@@ -1,6 +1,11 @@
 # Modeling with AWS CloudFormation
```

Comment:
I wonder if it would look less overwhelming for folks if we split into two pages?
- `Developing > Additional AWS Resources > Additional Workload Resources`
- `Developing > Additional AWS Resources > Additional Environment Resources`
- `Developing > Additional AWS Resources > Uploading Local Artifacts`

Reply:
What do you think of this?
- `Developing > Additional AWS Resources > Modeling with AWS CloudFormation > Additional Workload Resources`
- `Developing > Additional AWS Resources > Modeling with AWS CloudFormation > Additional Environment Resources`
- `Developing > Additional AWS Resources > Uploading Local Artifacts`

The hierarchy you proposed LGTM too. Just trying to preserve the CloudFormation keyword